graph-proxy/argo-workflows-openapi/build.rs

```
@@ -12,7 +13,23 @@ fn main() {
         .unwrap()
         .text()
         .unwrap();
-    let schema = serde_json::from_str::<schemars::schema::RootSchema>(&raw_schema).unwrap();
+    let mut schema: RootSchema = serde_json::from_str(&raw_schema).unwrap();
+
+    if let Some(Schema::Object(ref mut pod_gc)) = schema
+        .definitions
+        .get_mut("io.argoproj.workflow.v1alpha1.PodGC")
+    {
+        if let Some(pod_gc_object) = &mut pod_gc.object {
```

Should this fail silently if the structure changes? Or should we just `unwrap`?